gateway/docs/cluster-migration/onlineboutique-frontend-manifests.yaml

```diff
@@ -0,0 +1,109 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+##################################################
+# Referenced by [Link TBD]
+##################################################
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: frontend
+spec:
+  selector:
+    matchLabels:
+      app: frontend
+  template:
+    metadata:
+      labels:
+        app: frontend
+    spec:
+      serviceAccountName: default
+      containers:
+        - name: server
+          image: gcr.io/google-samples/microservices-demo/frontend:v0.3.5
+          ports:
+          - containerPort: 8080
+          readinessProbe:
+            initialDelaySeconds: 10
+            httpGet:
+              path: "/_healthz"
```

Comment:
If i understand it correctly from the all the microservices of the online boutique this is the only one which is exposed to the outside ? If that's the case , don't you need a BackendConfig object to customize the HealthCheck Path?

Reply:
Currently the BackendConfig resource is not supported for multi-cluster GatewayClasses (https://cloud.google.com/kubernetes-engine/docs/how-to/deploying-gateways#limitations) which we are using in the reference guide consuming these manifests. The app responds on port 80 for health as well, so this implementation does work in its current form. Happy to update with BackendConfig when it is supported for multi-cluster GatewayClasses if that works.